.goreleaser.docker.yaml

```
@@ -280,6 +303,52 @@ dockers:
       - '--label=org.opencontainers.image.title=jvs-ui'
       - '--label=org.opencontainers.image.version={{ .Version }}'
 
+  -
+    ids:
+    - jvsctl
+    use: 'buildx'
+    goos: 'linux'
+    goarch: 'amd64'
+    image_templates:
+      - '{{ .Env.CONTAINER_REGISTRY }}/prober:{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}-amd64'
+    dockerfile: ./prober/prober.dockerfile
+    extra_files:
+    - ./prober/prober.sh
+    build_flag_templates:
+      - '--platform=linux/amd64'
+      - '--pull'
+      - '--label=org.opencontainers.image.created={{ .CommitTimestamp }}'
+      - '--label=org.opencontainers.image.description=Prober service for jvs'
```

Not really a service
```suggestion
      - '--label=org.opencontainers.image.description=Prober for jvs'
```